### PR TITLE
back_to_roots

### PR DESCRIPTION
**Pull request by xanderrootslayer**
#### Summary
SUMMARY: Features "Adding three new live plants; Chicory, Salsify, and an unidentified wild root. Also updated the description for Dahlia Roots."

#### Purpose of change
"There have been many complaints that Wild Vegetables lack enough calories to live off of. Here are some thick, starchy roots to roast over the fire."

#### Describe the solution
"Handfuls of Salsify are full of calories and minerals. Wild Roots are as nutritious as carrots, but slightly dangerous. Chicory is... not coffee. It's very nutty though."

#### Describe alternatives you've considered
"I almost made a general "Baked Root" that Dahlia, Wild Root and Salsify all cook into but that was... too finicky for my tastes."

#### Additional context
"Had to edit regional map settings for this. Had to admit, toying with that file makes me nervous."

### CODE REVIEW

File: data/json/items/comestibles/raw_veggy.json [hunk 1]
```diff
@@ -264,11 +264,67 @@
...
+    "spoils_in": "30 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 43,
+    "description": "A modest blue flower. The flower's roots are very bitter, but they're not meant to be eaten raw.",
+    "price": 110,
+    "material": "veggy",
+    "volume": 1,
+    "fun": -8,
+    "vitamins": [ [ "vitC", 5 ], [ "iron", 3 ], [ "calcium", 2 ] ]
```
Comment (anothersimulacrum) [inline data/json/items/comestibles/raw_veggy.json]:
```suggestion
    "vitamins": [ [ "vitC", 5 ], [ "iron", 3 ], [ "calcium", 2 ] ],
```

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -1002,6 +1002,25 @@
     "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
     "components": [ [ [ "dahlia_root", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "salsify_baked",
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_food.json]: This seems like an easy recipe, perhaps add `"autolearn": true`, and bump down the level the books teach you it?

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -1316,6 +1335,27 @@
     "tools": [ [ [ "water_boiling_heat", 2, "LIST" ] ] ],
     "components": [ [ [ "raw_dandelion", 1 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "chicory_coffee",
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_food.json]: Same here

**Comment by anothersimulacrum** [comment]
You need to lint the files you touched.

File: data/json/recipes/recipe_food.json (hunk 2 above)
Comment (xanderrootslayer) [reply data/json/recipes/recipe_food.json]: Come to think of it, why do baked Dahlia Roots require a cookbook to understand?

File: data/json/items/comestibles/drink.json [hunk 4]
```diff
@@ -225,6 +225,27 @@
...
     "fun": -5
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "chicory_coffee",
+    "name": "chicory brew",
+    "name_plural": "chicory brew",
+    "weight": 252,
+    "color": "brown",
+    "addiction_type": "caffeine",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/drink.json]: AFAIK, there is no caffeine in chicory.

**Comment by KorGgenT** [comment]
you have an invalid summary line. please use the format described in the commented section of the template.

**Pull request merged**